app/models/user/reputation_period.rb

```diff
@@ -1,7 +1,7 @@
 class User::ReputationPeriod < ApplicationRecord
   enum period: { forever: 0, year: 1, month: 2, week: 3 }
   enum about: { everything: 0, track: 1 }, _prefix: true
-  enum category: { any: 0, building: 1, maintaining: 2, authoring: 3, mentoring: 4 }, _suffix: true
+  enum category: { any: 0, building: 1, maintaining: 2, authoring: 3, mentoring: 4, other: 5 }, _suffix: true
```

Comment:
I've added an `other` category. Do we want that or something else?

Reply:
I don't think we need this at all.

Reply:
Interesting. Thanks. This bit is quite complex in terms of what is going on, so shout if you want me to look with you.

Reply:
This is due to the `MarkForNewToken` class not being able to deal with the category. I've changed the code to exclude the `:misc` category, but want to double check if that is what we want. It depends on whether or not we want to show misc reputation on the contributors page:

![image](https://user-images.githubusercontent.com/135246/157630633-ec0af905-ca36-4187-8839-49bc3d1ca5c2.png)